apps/bors_frontend/config/config.exs

```diff
@@ -9,11 +9,11 @@ config :bors_frontend, ecto_repos: []
 
 # General application configuration
 config :bors_frontend, BorsNG,
-  activation_phrase: "bors r+",
-  activation_by_phrase: "bors r=",
-  deactivation_phrase: "bors r-",
+  activation_phrases: ["bors r+", "bors +r"],
+  activation_by_phrases: ["bors r="],
```

Comment:
The `=` sign connects the nickname with the `r`. The `activation_by_phrases` allow me to type in something like "r=Macarse" (I'm typing the command, but you get recorded as the reviewer). "=rMacarse" is dumb.